docs/home/whats-new-2-18.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Version 2.18
+summary: This article describes new features and functionality in the version.
+authors:
+    - Jamie Weider
+date: 2024-June-14
+---
+
+## Release Content - June 30, 2024
+
+* [Deprecation notifications](#deprecation-notifications)
+* [Breaking changes](#breaking-changes)
+
+### Researcher
+
+#### Jobs, Workloads, and Workspaces
+
+* <!-- TODO ADD LINK TO DOC Run-14732/Run-14733 Add backoff limit to workspace & standard training -->Added backoff limit functionality to Training and Workspace workloads. The backoff limit is the maximum number of retry attempts for failed workloads. After reaching the limit, the workload's status will change to `Failed`.
+
+* <!-- RUN-16830/RUN-16831 - Graphs & special metrics for inference -->Added new graphs for *Inference* workloads. The new graphs provide more information for *Inference* workloads to help analyze performance of the workloads. For more information, see [Workloads View](../admin/workloads/README.md#workloads-view).
+
```

Comment:
The fact that there is a default is new. @liohill , please have a look, it is not correct + we need to communicate it to CSM and validate if a FAQ is needed in order for them to share with customers.